test/support/left_join_schemas.ex

```diff
@@ -7,6 +7,7 @@ defmodule TrunkSchema do
   schema "trunk_schema" do
     field :child, :integer
     field :child_name, :string, virtual: true
+    field :computed, :boolean, virtual: true
```

Comment:
I dig it, adding the `?` should help readability.

RE: reused of the schema, I mostly just reused it because it was there and adding a virtual field doesn't require more migrations. Also I didn't want too add other changes like renaming the table to match the module just for a fix.

Reply:
for context I actually updated the field type to `:integer` as it made the reasoning  of the test easier.

Reply:
Perf, that does help, ty.